velox/docs/functions/spark/regexp.rst

```diff
@@ -74,6 +84,6 @@ See https://github.com/google/re2/wiki/Syntax for more information.
 
         SELECT regexp_replace('Hello, World!', 'l', 'L', 6); -- 'Hello, WorLd!'
 
-        SELECT regexp_replace('Hello, World!', 'l', 'L', -5); -- 'Hello, World!'
+        SELECT regexp_replace('Hello, World!', 'l', 'L', 5); -- 'Hello, World!'
 
-        SELECT regexp_replace('Hello, World!', 'l', 'L', 100); -- ERROR: Position exceeds string length.
+        SELECT regexp_replace('Hello, World!', 'l', 'L', 100); -- 'Hello, World!
```

Comment:
> 'Hello, World!

Are we missing closing quote?

Reply:
Sorry to hear that weren't feeling okay, I hope you're doing better now.

And yes, we are. I will make sure this is fixed.